docs/en_US/guides/sideloading-apps.md

```diff
@@ -81,18 +81,18 @@ AltServer is only compatible with iOS 12.2 and newer.
 
 ## Resigning Apps
 
-### Resigning using Reprovision Reborn
+### Resigning using ReProvision Reborn
 
 ::: tip
 
-ReProvision Reborn is compatible with iOS 9 and newer
+ReProvision Reborn is compatible with iOS 9 and newer.
 
 :::
 
 1. Add the Havoc Repo to your preferred package manager ([havoc.app](https://havoc.app/))
-1. Install Reprovison Reborn
-1. Open Reprovision Reborn and follow all onscreen prompts
-    - You will be asked to enter your AppleID. This is only sent to Apple and no one else.
+1. Install ReProvison Reborn
```

Comment:
So somehow this has been missed on both, but this is a typo and should be ReProvision